Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -164,5 +178,20 @@ public void updateMotorTelemetry() {
         leftDriveSlave.updateTelemetryProperties();
         rightDrive.updateTelemetryProperties();
         rightDriveSlave.updateTelemetryProperties();
+        
+        leftDistanceProp.set(leftDrive.getPosition());
+        rightDistanceProp.set(rightDrive.getPosition());
+    }
+    
+    public double getDistance() {
+        return convertToInches(leftDrive.getPosition() - startPosition.get());
```

Comment:
Indeed we were only getting values for one of the encoders and it's really tricky to get at the electronics on the summer bot to verify it was really there on the other side and plugged in etc.. As you said we'll want both sides averaged together on the final bot but I imagine we'll be using the summer bot for drive testing at least for the next few meetings. Maybe let's have the code to handle both there but commented out?